app/controllers/foreman_ansible/concerns/hostgroups_controller_extensions.rb

```diff
@@ -0,0 +1,29 @@
+module ForemanAnsible
+  module Concerns
+    # Extra methods to enforce Ansible roles on a host or multiple hosts
+    module HostgroupsControllerExtensions
+      extend ActiveSupport::Concern
+      include ForemanTasks::Triggers
+
+      def play_roles
```

Comment:
If we're going to use the API for this - mind to extend the api/v2/hostgroups_controller instead of the regular GET action? Other than that it looks great